binding-library/java/pom.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.microsoft.azure.functions</groupId>
+    <artifactId>azure-functions-java-library-signalr</artifactId>
+    <version>1.0-SNAPSHOT</version>
+    <packaging>jar</packaging>
+
+    <name>Microsoft Azure Functions Java SignalR Types</name>
+    <description>This package contains all Java interfaces and annotations to interact with Microsoft Azure functions runtime for SignalR Service.</description>
+    <url>https://docs.microsoft.com/en-us/azure/azure-functions/functions-bindings-signalr-service</url>
+    <organization>
+        <name>Microsoft Azure</name>
+        <url>https://azure.microsoft.com</url>
+    </organization>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <azure.functions.java.library.version>1.3.0-SNAPSHOT</azure.functions.java.library.version>
```

Comment:
>1.3.0-SNAPSHOT [](start = 46, length = 14)

Will update to release version(should be 1.3.0) after library published and before checked-in.

Reply:
It would be good to release a SNAPSHOT version first to enable E2E testing before releasing the official version. This is the same release process we follow of the library

Reply:
Thanks for the advise. Will do.

---
In reply to: [259939957](https://github.com/Azure/azure-functions-signalrservice-extension/pull/48#discussion_r259939957) [](ancestors = 259939957)